ui/mirage/models/project.ts

```diff
@@ -1,9 +1,9 @@
 import { Model, hasMany } from 'ember-cli-mirage';
-import { Project, Ref } from 'waypoint-pb';
+import { Project, Ref, Variable } from 'waypoint-pb';
```

Comment:
Can we remove this unused import?

```suggestion
import { Project, Ref } from 'waypoint-pb';
```